### ISSUE TITLE
Highlight First Letter of Two-letter combination

### ISSUE THREAD

**Issue by thorivarnj** [croian/mouseless-issues#9]
### Problem Description

Currently all letters are weighted equally, making it hard to see which key to press first in some cases. One example where this gets especially confusing is where the key combinations line up, in my case the combination `FG | GG` are next to each other.

### Proposed Solution

My proposed solution is to make the first letter of the combination weighted heavier than the second, drawing attention to which key to press first. This will make it even more intuitive at a glance, and can possibly make it even more intuitive to use.

**Comment by croian** [comment]
Excellent idea.  Will add this.  I'm thinking both size differentiation and RGBA differentiation could be helpful.

**Comment by shaunpatel** [comment]
On a related note, you can also dynamically highlight the cell border as soon as the first letter is pressed to reaffirm the motion.

**Comment by iminlikewithyou** [comment]
i think lowering the opacity of the second letter may be better for me (i dont think it would look good to weight the first letter) so i hope it's in the settings to change the options for the first and second letter, but i do really really like this

**Comment by thorivarnj** [comment]
I agree @iminlikewithyou. I think both options could work, as they ultimately solve the same issue
